pydoctor/test/test_epydoc2stan.py

```diff
@@ -86,16 +86,31 @@ def test_xref_link_not_found() -> None:
 
 def test_xref_link_same_page() -> None:
     """A linked name that is documented on the same page is linked using only
-    a fragment as the URL.
+    a fragment as the URL. But that does not happend in summaries. 
     """
-    mod = fromText('''
+    src='''
```

Comment:
We might want to enable `black` to create a common style for the code formatting :)
```suggestion
    src = '''
```